### DISCUSSION TITLE
Jellyfin metadata issues

### DISCUSSION THREAD

**Discussion by kabatp** [jellyfin/jellyfin#12923] [Q&A]
Jellyfin version: 10.10
OS: TrueNas Scale 24.04

As Jellyfin 10.10 supports delimiters for multiple artists, I tagged all music files with ; as separator for multiple artists in ALBUM and ALBUMARTISTS tags. I used mp3tag for this and I can see it successfully changed in mp3Tag, Pickard and Windows properties

![{2780E8F9-BBF0-479F-927E-14A85487D244}](https://github.com/user-attachments/assets/885c672b-24ad-470b-bd73-291a6cb0a7cb)

![{C848E61F-9EBC-4250-9869-EEC4B5F0D9A9}](https://github.com/user-attachments/assets/4a78d089-2101-4758-97b3-dbf8764e0204)

But Jellyfin sees it in the old way: 
![{F5725388-C1EF-4F63-916E-90407426BB7F}](https://github.com/user-attachments/assets/2124af12-a181-4401-9505-9bd287dd9250)

I already tried to:
Rescan the library
Remove the music file from the music library and re-add it back
Remove the whole music library and re-add it back
Remove all metadata
Deleted metadata/artists folder and also cache folder
Restart the Jellyfin server
Restart the Jellyfin container

The delimiter update in 10.10 seems to work for most of the music files - as this error occurs in circa 25 out of 1345 artists, but there is no way for me to change these.

Even if I manage to change the metadata via web GUI:
![{03DDD846-5D1F-4FE6-91EF-C53B7D5E1196}](https://github.com/user-attachments/assets/78e66f2f-6661-4622-b02a-4e9a6f53a90b)

The old artist in Artist tab stays:
![{96BDB53D-CE9D-42E9-B3DD-0951642FE0AE}](https://github.com/user-attachments/assets/b6322061-9828-407d-a98a-5905adc7c734)

Showing that he appears on the album:
![{1D9A607E-3AC0-4FC6-A220-CA223BC81998}](https://github.com/user-attachments/assets/6075a4dd-5896-4338-b477-353d9d8c236c)

**Comment by kabatp** [comment]
I just tried a brand new instance of Jellyfin, with only access to the music library to speed up the setup.

I disabled all metadata providers, so only image providers are enabled. The issue changed slightly - no more blurred pictures or such, but the artists' naming issue persists, for example: 
![{2C3692FA-F6B1-45A0-9CCF-832F105C6C06}](https://github.com/user-attachments/assets/396bfe89-0d79-4769-a9ef-8b39f3a19c23)


The same artist appeared 3 times - before my tag clean up, the artist was stored in songs in the way shown in the picture, since then I changed the name of the folders and also tags to reflect the changed. Now all songs are stored in one folder with the correct name, all songs have the same ARTIST and ALBUMARTISTS tag.

Are there any more tags that I can change that I possibly forgot about and Jellyfin is using them?

**Comment by tuxpizza** [comment]
I just ran into similar issues myself after upgrading to 10.10.0 and wanted to check here before opening an issue
Quite frustrating as I can't add new music because the tagging is totally screwed up

In 10.9, it was changed so that the `Artist` tag now used the standard ID3v2 standard separator instead of ; where as the `Album Artist` tag still required ; to separate the artists.

Now with 10.10, both of these tags are completely broken, and Jellyfin combines the `Artist` tag into a single word if there are separated artists, and the `Album Artist` tag just keeps the ; and doesn't separate the artist.

I tried just about every separator possible, and no other separator would work.

It seems like going back to old songs and modifying the metadata and forcing a metadata refresh works, but when adding new music its totally broken.

Oh and Genre tag is partially broken and no longer works with the ; separator

**Comment by kabatp** [comment]
@tuxpizza you can change the delimiter to custom, same as to use standard or non-standatd artist tag. You go Dashboard - Libraries - Manage library - At the total bottom you can change the delimiters etc., then you need to rescan with replace metadata and it should work.

My issue is that when I change something and go with following steps
- Move the artist folder with music out of the server directory
- Change the tags and manually change nfo files (or delete them)
- Rescan the library (to be sure jellyfin knows they have been moved out)
- Delete folder with broken artists from config/metadata/artists
- Restart the server
- Move artist folder with music back
- Rescan the library

The same metada appears even tho there is no tag or nfo mentioning the broken artist

**Reply by tuxpizza** [reply]
Oh shit thanks, that must be new.... I totally didn't notice :saluting_face: 

Still seems to be a problem with new music and the standard separator (ID3v2)
